simulators/blocktests/testrunner/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Docker container spec defining and running a simple smoke test consisting of
+# starting a single client node and retrieving some data from it.
+FROM jfloff/alpine-python:2.7-onbuild
+
+# Install curl to allow controlling the simulator as well as to interact with the
+# client via HTTP-RPC.
+RUN apk add --update curl git && \
+  git clone --depth 1 https://github.com/ethereum/tests.git
+
+
+# Add the simulation controller
+ADD simulator.sh /simulator.sh
+RUN chmod +x /simulator.sh
+COPY *.py /
+
+ENTRYPOINT ["python","/simulator.py"]
```

Comment:
From the hive readme:

> The simulator definition(s) should reside in the simulators folder, each nested as <group>/<sim>, where <group> is a higher level collection of similar tests (e.g. dao-hard-fork), and <sim> is an individual simulator.

I'm not really sure what `blocktests/testrunner` hierarchy really means. Wouldn't it make more sense to have `consensus/block(tests)`? I.e. it would be nice to have all EF consensus tests groupped under one high level group (i.e. `consensus`), and I don't see a reason to have a further netsted `testrunner` folder in there.